stream_chat/async_chat/client.py

```diff
@@ -190,6 +190,29 @@ async def unflag_user(self, target_id: str, **options: Any) -> StreamResponse:
         data = {"target_user_id": target_id, **options}
         return await self.post("moderation/unflag", data=data)
 
+    async def query_flag_reports(self, **options: Any) -> StreamResponse:
+        """
+        Note: Do not use this.
+        It is present for internal usage only.
+        This function can, and will, break and/or be removed at any point in time.
+        """
+        return await self.post("moderation/reports", data=options)
+
+    async def review_flag_report(
+        self, report_id: int, review_result: str, user_id: str, **details: Any
+    ) -> StreamResponse:
+        """
+        Note: Do not use this.
```

Comment:
maybe prefix this method with an underscore to signify it is private

Reply:
Updated, please check.